smartmin/templates/smartmin/field.html

```diff
@@ -6,7 +6,7 @@
 {% if form_field and form_field.is_hidden %}
 {{ form_field }}
 {% else %}
-{% if form_field %}
+{% if form_field != None %}
```

Comment:
Actually our next release of Smartmin would be for 1.9, 1.10 and 1.11 so we can use that